### PR TITLE
chore: prepare 2.4.0 release

### PR DESCRIPTION
**Pull request by Mat001**
## Summary
Changelog for 2.4.0 (decide-api) release

### CODE REVIEW

**Review by Mat001 — COMMENTED** [review]
@jaeopt What do you suggest I write instead of the set of new APIs. We only have one new endpoint `/decide`. The rest are APIs based on GO-SDK

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,17 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
-    - SetAttribute
-    - Decide
-    - DecideAll
-    - DecideForKeys
-    - TrackEvent
-
-For details, refer to our documentation page: https://docs.developers.optimizely.com/full-stack/v4.0/docs/optimizely-agent.
+For details about this Agent release, refer to our documentation page: https://docs.developers.optimizely.com/full-stack/v4.0/docs/optimizely-agent.  
+Under the hood Agent uses Go SDK which implements Decide APIs. If interested to learn more about them see [Go SDK v1.6.0](https://github.com/optimizely/go-sdk/tree/v1.6.0).
```
Comment (jaeopt) [inline CHANGELOG.md:13]:
```suggestion
Upgrade to use [Go SDK v1.6.0](https://github.com/optimizely/go-sdk/tree/v1.6.0). This adds support for OptimizelyDecision.
```

**Review by jaeopt — CHANGES_REQUESTED** [review]
Change go sdk message to be consistent with prev versions

**Comment by Mat001** [comment]
@msohailhussain Can I ask you also to review pls

File: go.sum [hunk 2]
```diff
@@ -93,6 +93,8 @@ github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRW
 github.com/oklog/ulid v1.3.1/go.mod h1:CirwcVhetQ6Lv90oh/F+FBtV6XMibvdAFo93nm5qn4U=
 github.com/optimizely/go-sdk v1.6.1-0.20210226222257-68aca7d10f77 h1:YGHuD2FA1qPkRqDSKJwuw5pGQmip8S6hOBcRPcYKntE=
```
Comment (yasirfolio3) [inline go.sum:94]:
```suggestion
```

File: go.sum [hunk 3]
```diff
@@ -93,6 +93,8 @@ github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRW
...
 github.com/optimizely/go-sdk v1.6.1-0.20210226222257-68aca7d10f77/go.mod h1:1uinGREH+AdijSRw3qitWkvIna1e/ZGN5eymNYPjw1A=
```
Comment (yasirfolio3) [inline go.sum:95]:
```suggestion
```

**Review by yasirfolio3 — CHANGES_REQUESTED** [review]
Have suggested some changes, please have a look.

**Comment by Mat001** [comment]
@jaeopt @yasirfolio3 Can you review/approve?
Your requested changes have been addressed.

**Review by yasirfolio3 — APPROVED** [review]
LGTM

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**